rest_framework/renderers.py

```diff
@@ -64,15 +70,16 @@ class JSONRenderer(BaseRenderer):
     # We don't set a charset because JSON is a binary encoding,
     # that can be encoded as utf-8, utf-16 or utf-32.
     # See: http://www.ietf.org/rfc/rfc4627.txt
-    # Also: http://lucumr.pocoo.org/2013/7/19/application-mimetypes-and-encodings/
+    # http://lucumr.pocoo.org/2013/7/19/application-mimetypes-and-encodings/
```

Comment:
If/When the decision is made to follow a 79 character line length, the comments should be pulled over too.  But since that won't be happening in the short term, the comments will stay the same.